### PR TITLE
Identity server url fetched from env variable

### PR DESCRIPTION
**Pull request by Thorrik58**
# Use env for identity server url

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/service-portal/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
-export const environment = {
+export default {
   production: true,
+  identityServer: {
+    IDENTITY_SERVER_BASE_URL:: process.env.IDENTITY_SERVER_BASE_URL,
```
Comment (obmagnusson) [inline apps/service-portal/src/environments/environment.prod.ts:4]: double ::

**Review by obmagnusson — CHANGES_REQUESTED** [review]

File: apps/service-portal/src/utils/userManager.ts [hunk 2]
```diff
@@ -4,9 +4,10 @@ import {
   InMemoryWebStorage,
   UserManagerSettings,
 } from 'oidc-client'
+import { environment } from '../environments'
 
 const settings: UserManagerSettings = {
-  authority: 'https://siidentityserverweb20200805020732.azurewebsites.net/',
+  authority: environment.identityServer.IDENTITY_SERVER_BASE_URL,
```
Comment (pshomov) [inline apps/service-portal/src/utils/userManager.ts:10]: I think this should be just fine, do you have the environment config replacement thing in workspace.json?
Also not sure if it matters but you had originally a trailing forward slash and now it is missing.

**Review by pshomov — COMMENTED** [review]
LGTM

File: apps/service-portal/src/utils/userManager.ts (hunk 2 above)
Comment (Thorrik58) [reply apps/service-portal/src/utils/userManager.ts:10]: @pshomov 
Yes we have the file replacement line in our workspace.json.

Ill double check the trailing slash. 

Trying to load this env variable locally from the environment file by pointing towards process.env.IDENTITY_SERVER_BASE_URL in environments.ts doesnt work. But maybe its just not working locally for once :P

Comment (pshomov) [reply apps/service-portal/src/utils/userManager.ts:10]: if you want to try it locally then you should rename your .prod file to be your default one and then try it. Alternatively you can `yarn build` your project with `--prod` flag to make it build a prod distro and run it directly.

**Review by obmagnusson — APPROVED** [review]
LGTM give it a try

**Pull request merged**